frontend/src/components/resourceMap/sources/definitions/NetworkSource.tsx

```diff
@@ -0,0 +1,167 @@
+import { Icon } from '@iconify/react';
+import { useMemo } from 'react';
+import Endpoints from '../../../../lib/k8s/endpoints';
+import Ingress, { IngressRule } from '../../../../lib/k8s/ingress';
+import IngressClass from '../../../../lib/k8s/ingressClass';
+import NetworkPolicy from '../../../../lib/k8s/networkpolicy';
+import Pod from '../../../../lib/k8s/pod';
+import Secret from '../../../../lib/k8s/secret';
+import Service from '../../../../lib/k8s/service';
+import { GraphEdge, GraphSource } from '../../graph/graphModel';
+import { getKindGroupColor, KubeIcon } from '../../kubeIcon/KubeIcon';
+import { makeKubeObjectNode, makeKubeToKubeEdge } from '../GraphSources';
+import { matchesSelector } from './WorkloadSource';
+
+const serviceSource: GraphSource = {
```

Comment:
The filenames make me think these are react components, but they are not AFAIK.

Reply:
changed to lowercase